### PR TITLE
Osa management

### PR DESCRIPTION
**Pull request by martindev727**

### CODE REVIEW

File: src/components/osa-management/osa-management.component.jsx [hunk 1]
```diff
@@ -1,9 +1,91 @@
...
-    return (<div>OSAManagement Page</div>)
+    return (
+      <div className="osa-management-page-wrapper">
+        <section>
+          <div className="primary-container">
+            <div className="content-container">
+              <div className="header-comment">
+                Management of Osa
+              </div>
+              <div className="content-header">
```
Comment (rphovley) [inline src/components/osa-management/osa-management.component.jsx]: Needs to be h1 tag (refer to this document in the future https://hoverstate.egnyte.com/fl/CYSPsEZalw#folder-link/Dev%20Delivery/SEO/Content%20Coding%20Directions for SEO optimizations. We just received this, Eric's branch that just got merged in kicked off us being more aware of html selectors for SEO)

File: src/components/osa-management/osa-management.component.jsx [hunk 2]
```diff
@@ -1,9 +1,91 @@
...
+              </div>
+            </div>
+          </div>
+        </section>
+        <section className="airway-pressure">
+          <div className="airway-pressure-container">
+            <img src={functionalYawn} alt="Continuous positive airway pressure" className="airway-pressure-img"/>
+            <div className="airway-pressure-text-container">
+              <div className="airway-pressure-text-content">
+                <div className="text-content-title">Continuous positive airway pressure (CPAP) is the standard of care for managing moderate to severe OSA<sup>1,2</sup></div>
```
Comment (rphovley) [inline src/components/osa-management/osa-management.component.jsx]: H2 tag

File: src/components/osa-management/osa-management.component.jsx [hunk 3]
```diff
@@ -1,9 +1,91 @@
...
+                  <li><span>Positive airway pressure devices<sup>1,2</sup></span></li>
+                </ul>
+                <p className="content-info">Surgical options include maxilla-mandibular advancement (MMA), tonsillectomy and/or adenoidectomy, uvulopalatopharyngoplasty, and hypoglossal nerve stimulation.<sup>1-6</sup></p>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section className="airway-pressure">
+          <div className="airway-pressure-container">
+            <img src={functionalYawn} alt="Continuous positive airway pressure" className="airway-pressure-img"/>
```
Comment (rphovley) [inline src/components/osa-management/osa-management.component.jsx]: alt="Sleeping while using a CPAP machine"

Also, have you checked with Ani Zargaryan for the stock photo on this page? It looks like you are using a different photo for now?

File: src/components/osa-management/osa-management.component.jsx [hunk 4]
```diff
@@ -1,9 +1,91 @@
 import React, { Component } from 'react'
 import './osa-management.component.css'
+import LearnMoreComponent from '../learn-more/learn-more.component'
+import EmailSignupComponent from '../email_signup/email_signup.component'
+import ReferenceComponent from '../references/reference.component'
+
+import functionalYawn from '../../assets/images/functional-yawn.png'
```
Comment (rphovley) [inline src/components/osa-management/osa-management.component.jsx]: filename for the primary on this page is to be `sleeping-cpap-machine.svg` when you get it from Ani

File: src/components/osa-management/osa-management.component.jsx [hunk 5]
```diff
@@ -1,9 +1,91 @@
...
+        { content: 'Zhu Y, Fenik P, Zhan G, Xin R, Veasey SC. Degeneration in arousal neurons in chronic sleep disruption modeling sleep apnea. ', book: 'Front Neurol. ', info: '2015;6:109.' },
+        { content: 'Joo EY, Tae WS, Lee MJ, et al. Reduced brain gray matter concentration in patients with obstructive sleep apnea syndrome. ', book: 'Sleep. ', info: '2010;33(2):235-241.' },
+        { content: 'Lal C, Strange C, Bachman D. Neurocognitive impairment in obstructive sleep apnea. ', book: 'Chest. ', info: '2012;141(6):1601-1610.' },
+        { content: 'Xiong Y, Zhou XJ, Nisi RA, et al. Brain white matter changes in CPAP-treated obstructive sleep apnea patients with residual sleepiness.', book: 'J Magn Reson Imaging. ', info: '2017;45(5):1371-1378. ' }
+      ],
+      esPersist: {
+        title: 'Can ES persist in OSA, even with CPAP compliance?',
+        subtitle: 'Check out the statistics.',
+        link: '/',
+        image: persistIcon
```
Comment (rphovley) [inline src/components/osa-management/osa-management.component.jsx]: add alt attribute  `alt='Excessive sleepiness (ES) in OSA - graph icon'`
fix link to go to correct page using new sitemap navigation from Eric's branch

File: src/components/osa-management/osa-management.component.jsx [hunk 6]
```diff
@@ -1,9 +1,91 @@
...
+      },
+      emergingScience: {
+        title: 'See the emerging science that may explain ES in OSA.',
+        subtitle: '',
+        link: '/',
+        image: emergingIcon
```
Comment (rphovley) [inline src/components/osa-management/osa-management.component.jsx]: add alt attribute `alt: 'Excessive sleepiness (ES) in OSA - microscope icon'`
Fix link to go to correct page using new sitmap navigation from Eric's branch

File: src/components/osa-management/osa-management.component.jsx [hunk 7]
```diff
@@ -1,9 +1,91 @@
 import React, { Component } from 'react'
 import './osa-management.component.css'
+import LearnMoreComponent from '../learn-more/learn-more.component'
+import EmailSignupComponent from '../email_signup/email_signup.component'
+import ReferenceComponent from '../references/reference.component'
+
+import functionalYawn from '../../assets/images/functional-yawn.png'
+import persistIcon from '../../assets/images/persist.png'
+import emergingIcon from '../../assets/images/emerging.png'
```
Comment (rphovley) [inline src/components/osa-management/osa-management.component.jsx]: rename persist to be 
`excessive-sleepiness-osa-graph-icon.png`
rename emerging to be
`excessive-sleepiness-osa-microscope-icon.png`

**Comment by rphovley** [comment]
There's quite a few changes to be made to this branch since Eric merged in his branch with SEO updates. Rebase onto master and apply SEO optimization changes and a few small other things I mention in the code and we'll get this merged in.

File: src/components/SiteLink.jsx [hunk 8]
```diff
@@ -7,7 +7,7 @@ import SiteMap from './SiteMap'
 class SiteLinkComponent extends Component {
   constructor(props) {
     super(props)
-
+    console.log(props.to)
```
Comment (rphovley) [inline src/components/SiteLink.jsx]: Remove console log :)

Comment (rphovley) [reply src/components/SiteLink.jsx]: Everything else looks good

**Comment by martindev727** [comment]
@rphovley 
I removed `console.log`

**Pull request merged**